package.json

```diff
@@ -45,10 +45,10 @@
   },
   "dependencies": {
     "compressible": "~2.0.6",
-    "debug": "*",
+    "debug": "3.1.0",
```

Comment:
we can use `^` for all these dependencies.

Reply:
@dead-horse done. :-)